test/decompiler/reference/jak2/engine/game/task/task-arrow_REF.gc

```diff
@@ -0,0 +1,368 @@
+;;-*-Lisp-*-
+(in-package goal)
+
+;; failed to figure out what this is:
+(defskelgroup skel-task-arrow ctywide-arrow 0 2
```

Comment:
Ah yeah, I think you just need to update the actual goal source file for this one.

Reply:
Is it not possible to eventually improve the decompiler to handle these types of `defskelgroup`s?

Reply:
It does. Check the REF file, it's detected.

Reply:
ah k, i misinterpreted